Review

KznRkjp opened the pull request.

KznRkjp committed:
Initial commit

KznRkjp committed:
Initial commit

KznRkjp committed:
added some modules

KznRkjp committed:
123

KznRkjp committed:
added buildinfo and flags

KznRkjp committed:
sync

KznRkjp committed:
new shutdown message

KznRkjp committed:
suffer

KznRkjp committed:
databse is working

KznRkjp committed:
database

KznRkjp committed:
прокрастинация с инфо

KznRkjp committed:
bulding line by line

KznRkjp committed:
users are able to login now

KznRkjp committed:
some comments

KznRkjp committed:
bugfix

KznRkjp committed:
added logopass upload

KznRkjp committed:
11

KznRkjp committed:
client

KznRkjp committed:
before pull request

KznRkjp committed:
+ readme.md

KznRkjp committed:
+ delete

KznRkjp committed:
+ put

KznRkjp committed:
client - the beginning

KznRkjp committed:
client + cookie

KznRkjp committed:
cookies continued

KznRkjp committed:
+ client Registeruser

KznRkjp committed:
+ little touch of config

KznRkjp committed:
+ LoginUser

KznRkjp committed:
6pm sync

KznRkjp committed:
fixed data aquire

KznRkjp committed:
added pretty table

KznRkjp committed:
cleanup

KznRkjp committed:
modules upgrade

KznRkjp committed:
123

KznRkjp committed:
+ bin

KznRkjp committed:
it's going and going

KznRkjp committed:
client can add logopass record now

KznRkjp committed:
crypto is in the house

KznRkjp committed:
не стреляйте в пианиста

KznRkjp committed:
last push for tonight

KznRkjp committed:
some more progress

KznRkjp committed:
sync

KznRkjp committed:
123

KznRkjp committed:
Merge pull request #1 from KznRkjp/finals

в процессе

KznRkjp committed:
Merge branch 'empty' into review

KznRkjp committed:
going and going

KznRkjp committed:
bug fix

KznRkjp committed:
hid password field

KznRkjp committed:
6pm sync

s-shpak commented on bin/server/mac/go-keeper-server:
Этого не должно быть в репозитории

KznRkjp replied:
Я  исходил из:

> Дополнительные требования:
> клиент должен распространяться в виде CLI-приложения с возможностью запуска на платформах Windows, Linux и Mac OS;
Удалю потом )

KznRkjp committed:
need to save while it works

KznRkjp committed:
modified:   internal/app/delete_data.go
	modified:   internal/clientapp/delete.go
	modified:   internal/clientapp/interfacebc.go
	modified:   internal/clientapp/interfacelp.go
	new file:   internal/clientapp/put.go
	modified:   internal/database/deletedata.go

KznRkjp committed:
+ readme

KznRkjp committed:
added bin and modified readme

KznRkjp committed:
1

KznRkjp committed:
2

KznRkjp committed:
login password edit +

KznRkjp committed:
123

KznRkjp committed:
6 pm sync

KznRkjp committed:
refactoring

KznRkjp committed:
bank card edit +

KznRkjp committed:
fixed txt

KznRkjp committed:
bugfix

KznRkjp committed:
alfa is ready

KznRkjp committed:
removed bin

KznRkjp committed:
# modified:   .gitignore

KznRkjp committed:
asciinema server

KznRkjp committed:
asciinema client

KznRkjp committed:
misstype errors

KznRkjp committed:
+ comments for some functions

KznRkjp committed:
1
